src/do_command.c

```diff
@@ -589,6 +589,12 @@ static int child_process(entry * e, char **jobenv) {
 				Debug(DPROC, (", dumped core"));
 			Debug(DPROC, ("\n"));
 	}
+	if ((e->flags & DONT_LOG) == 0) {
+		char *x = mkprints((u_char *) e->cmd, strlen(e->cmd));
```

Comment:
Why are you calling mkprints() here when just calling log_it(..., e->cmd, ..) should work fine?

Reply:
Actually no, that's a nonsense, what I wrote. :smile: